.circleci/config.yml

```diff
@@ -82,207 +84,312 @@ orbs:
+        - attach_workspace:
+            at: /home/circleci
+        - restore_cache: # If exist, restore dependencies libs download cache, from previous pipeline execution.
+            name: Restore SonarQube Scanner
+            keys:
+              - sonar-scanner-3.3.0.1492-linux
+        # Sonar steps
+        - run:
+            name: Download SonarQube Scanner
+            command: |
+              chmod +x /home/circleci/cassandra-k8s-operator/install_sonar-scanner.sh
+              ./.circleci/install_sonar-scanner.sh
+        - save_cache: # Cache sonar download
+            name: Cache SonarQube Scanner
+            key: sonar-scanner-3.3.0.1492-linux
+            paths:
+              - sonar-scanner-3.3.0.1492-linux
+              - /home/circleci/cassandra-k8s-operator/.sonar
+        - run: # Run sonar scan
+            name: Sonarcloud Scan
+            command: |
+              ./sonar-scanner-3.3.0.1492-linux/bin/sonar-scanner \
+              -Dsonar.projectName=<< parameters.operatorName >> \
+              -Dsonar.projectKey=${SONAR_PROJECT} \
+              -Dsonar.login=${SONAR_TOKEN} \
+              -Dsonar.branch.name=${CIRCLE_BRANCH} \
+              -Dsonar.organization=${SONAR_ORGANISATION} \
+              -Dsonar.host.url=https://sonarcloud.io \
+              -Dsonar.projectBaseDir=/home/circleci/<< parameters.operatorDir >> \
+              -Dsonar.sources=. \
+              -Dsonar.sources.inclusions="**/**.go" \
+              -Dsonar.exclusions="**/*_test.go,**/vendor/**,**/sonar-scanner-3.3.0.1492-linux/**,**docs/**,**/**/zz_generated*" \
+              -Dsonar.coverage.exclusions="**/vendor/**,**/test/**,**docs/**" \
+              -Dsonar.tests=. \
+              -Dsonar.language=go \
+              -Dsonar.sourceEncoding=UTF-8 \
+              -Dsonar.test.inclusions="**/**_test.go" \
+              -Dsonar.test.exclusions="**/vendor/**" \
+              -Dsonar.go.coverage.reportPaths=coverage.out \
+              -Dsonar.go.tests.reportPaths=test-report.out \
+              -Dsonar.coverage.dtdVerification=false \
+              -Dsonar.log.level=INFO
+              cat -n .scannerwork/report-task.txt
+        # Artifacts definitions
+        - store_test_results: # Store sonar scan reports
+            path: reports
+
+# Workflow definition
 workflows:
   version: 2
-
-  master:
+  # CassKop operator workflow.
+  ci-casskop-operator:
     jobs:
+      - operator/generate:
+          name: generate-casskop
+          filters:
+            tags:
+              only: /.*/
 
-      - build:
+      - operator/unit-test:
+          name: unit-test-casskop
+          requires:
+            - generate-casskop
           filters:
             tags:
               only: /.*/
 
-      - unit-test:
+      - operator/build:
+          name: build-casskop
           requires:
-            - build
+            - unit-test-casskop
           filters:
             tags:
               only: /.*/
```

Comment:
@erdrix, Our CI config is getting too long and complicated. Use [anchors](https://confluence.atlassian.com/bitbucket/yaml-anchors-960154027.html) as much as possible to refactor it and avoid repeating filters, steps etc...

Reply:
Add some anchors to refactor yaml.